clang/include/clang/Basic/Features.def

```diff
@@ -307,6 +307,10 @@ EXTENSION(datasizeof, LangOpts.CPlusPlus)
 
 FEATURE(cxx_abi_relative_vtable, LangOpts.CPlusPlus && LangOpts.RelativeCXXABIVTables)
 
+// Type aware allocators
+FEATURE(cxx_type_aware_allocators, LangOpts.TypeAwareAllocators)
```

Comment:
Reminds me: Do we need/have a feature test macro for these?

Reply:
Oh, I don't believe I've ever made a feature test macro (everything I've worked on has been via direct `__has_feature()` checks, I'll look into existing macros to see how they're implemented/exposed)